packages/libs/eda/src/lib/core/components/variableSelectors/VariableCollectionSelectList.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useMemo } from 'react';
+import { CollectionVariableTreeNode, StudyEntity } from '../../types/study';
+import SingleSelect, {
+  ItemGroup,
+} from '@veupathdb/coreui/lib/components/inputs/SingleSelect';
+import { Item } from '@veupathdb/coreui/lib/components/inputs/checkboxes/CheckboxList';
+import { useStudyEntities } from '../../hooks/workspace';
+import { VariableCollectionDescriptor } from '../../types/variable';
+
+interface Props {
+  collectionPredicate?: (collection: CollectionVariableTreeNode) => boolean;
+  value?: VariableCollectionDescriptor;
+  onSelect: (value?: VariableCollectionDescriptor) => void;
+}
+
+export function VariableCollectionSelectList(props: Props) {
+  const { collectionPredicate, onSelect, value } = props;
+  const entities = useStudyEntities();
+
+  const items = useMemo(() => {
```

Comment:
Why get the collections here within the component, instead of passing in an array of collections? In the apps it looks like we already have `collections`, which is a filtered list of collections, no?

Reply:
I went back-and-forth on this, and ultimately decided on this, to be more consistent with other variable selectors. Generally speaking, it's probably better to pass things in, rather than look them up on React Context (hidden dependencies, and all).

At this point, it appears the only reason we are getting collections in the apps is to ensure the study has collections. Does mbio have any such studies?

Reply:
Ahh got it. Makes sense to me!

Mbio has no such studies. All the guts of mbio data are always in a collection so even in the future its very very unlikely we'll ever load data without a collection!

Reply:
Cool! Do you think it's okay to remove the current check we're doing?

The UX of the current code probably isn't great: if a study does not have any collections, the entire viz tab will show an error message. If we remove the check, instead they will see an empty `SingleSelect` list, with no way to run the compute, which doesn't seem _too_ bad.

Reply:
Yeah i agree, the ux after removing the check sounds better. It'd be disruptive enough to (hopefully) make a user contact us about it but not so disruptive that they would stop doing anything at all on the site!

I made an [issue](https://github.com/VEuPathDB/web-monorepo/issues/723) to improve the no-collection ux. I know i need to do t[his related one for correlation soon](https://github.com/VEuPathDB/EdaNewIssues/issues/731), so if it turns out i see an easy way to do both i will!

Reply:
oh i just read your final comment that you already removed the check. Awesome thank you!